src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,7 @@
+package wolox.training.exceptions;
+
+public class BookIdMismatchException extends RuntimeException {
```

Comment:
You can add a status code to the exceptions, check this out: https://www.baeldung.com/spring-response-status-exception 😄